AmazonChimeSDKDemo/AmazonChimeSDKDemo/JoiningViewController.swift

```diff
@@ -58,9 +62,16 @@ class JoiningViewController: UIViewController, UITextFieldDelegate {
         }
 
         // Audio Mode
-        var audioMode: AudioMode = .mono
-        if !audioSwitch.isOn {
+        var audioMode: AudioMode = .stereo48K
+        switch audioModeOptionsPicker.selectedRow(inComponent: 0) {
+        case 1:
+            audioMode = .mono48K
+        case 2:
+            audioMode = .mono16K
+        case 3:
             audioMode = .noAudio
+        default:
+            audioMode = .stereo48K
```

Comment:
Can we have these 2 picker selected check as separate private methods?

Reply:
Will do